wire-library/wire-gradle-plugin/src/test/kotlin/com/squareup/wire/gradle/WirePluginTest.kt

```diff
@@ -103,6 +103,18 @@ class WirePluginTest {
             .contains("src/test/projects/sourcepath-build-dir/build/generated/source/wire".withPlatformSlashes())
   }
 
+  @Test
+  fun requireTarget() {
+    val fixtureRoot = File("src/test/projects/require-target")
+
+    val result = gradleRunner.runFixture(fixtureRoot) { buildAndFail() }
+
+    val task = result.task(":generateProtos")
+    assertThat(task).isNull()
```

Comment:
I wonder why it would be null, isn't it set but failed? Do you know?

I was expecting something like this
```suggestion
    assertThat(result.task(":generateProtos")?.outcome).isEqualTo(TaskOutcome.FAILED)
```

Reply:
It should be null because we failed to apply the plugin and haven't started running the `generateProtos` task